content/about/governance/_index.md

```diff
@@ -26,98 +18,188 @@ While purpose of the association is to support research software engineering and
+      - cooperates with public authorities, organizations, companies and individuals
+      - may recognize significant achievements in the field of scientific software development and related fields
 
-    The association fulfills its purpose by
+    In order to support its activities, with the proper permissions from authorities, the association
+     - may organize paid events
+     - may accept grants, donations and wills
+     - may own goods and real property necessary for its operations
+     - may accept sponsorships
 
-    - maintaining a network of research software engineers and other similar specialists and representing that network to others
-    - hosting events for members and the surrounding community
-    - supporting training and professional development in related skills for members and the surrounding community
-    - recognizing significant contributions to research software engineering and related fields
-    - advocating for research software engineers and engineering
-    - other activities in the public interest promoting research software and scientific computing
+3. Membership
+    Any person who approves the purpose and the statutes of the association can become a member. The board approves members on request.
 
-   In order to support its activities, the association may accept and manage funds according to relevant permits and own property.
+    The board may remove a member who acts against the purpose of the association, violates the code of conduct of the association, or who has neglected to pay their membership fee. Before removal, the member has the right to know the reason for their removal and express any objections in writing.
 
 4. Membership fee:
-    The annual meeting decides whether an annual membership fee is collected and the size of the fee.
+    The annual meeting decides size of the membership fee and registration fee.
 
 5. Board:
-    The association's affairs are handled by the board, which has a chair and 3-7 other regular members, as well as 0-8 deputies, all of whom are appointed at the annual meeting. The term of office of the Board is the time between the annual meetings.
+    The association's affairs are handled by the board, which has a chair and 3-7 other regular members, as well as 0-8 deputies, all of whom are appointed at the annual meeting. The term of office of the board is the time between the annual meetings.
+
+    The board chooses a vice their from amongst its members, and additionally a secretary, a treasurer and any additional necessary functionaries. Board meetings are convened by the chair, or the vice chair when the chair is unable, when they consider it necessary of at least half of the board members request it. The board has a quorum when at least half of its members, the chair and vice chair included, are present. Votes are resolved by strict majority. In the event of a tie, the chair has the casting vote, except in elections where ties are broken by lot.
```

Comment:
Shouldn't "vice their" be vice chair?

Reply:
"of at least half of the board members request it" -> ", or at least half of the board members request it"